source/Crew.cpp

```diff
@@ -0,0 +1,240 @@
+/* Crew.cpp
+Copyright (c) 2019 by Luke Arndt
+
+Endless Sky is free software: you can redistribute it and/or modify it under the
+terms of the GNU General Public License as published by the Free Software
+Foundation, either version 3 of the License, or (at your option) any later version.
+
+Endless Sky is distributed in the hope that it will be useful, but WITHOUT ANY
+WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
+PARTICULAR PURPOSE.  See the GNU General Public License for more details.
+*/
+
+#include "Crew.h"
+#include "GameData.h"
+
+using namespace std;
+
+// Load definition of a crew member
+void Crew::Load(const DataNode &node)
+{
+	// Set the name of this crew member so we know that we have loaded it.
+	if(node.Size() >= 2)
+		name = node.Token(1);
+	
+	// Set default values so that we don't have to specify every node.
+	avoidsEscorts = false;
+	avoidsFlagship = false;
+	isPaidSalaryWhileParked = false;
+	salary = 100;
+	minimumPerShip = 0;
+	populationPerMember = 0;
+	
+	for(const DataNode &child : node)
+	{
+		if(child.Size() >= 2)
+		{
+			if(child.Token(0) == "name")
+				name = child.Token(1);
+			else if(child.Token(0) == "salary")
+				if(child.Value(1) >= 0)
```

Comment:
Just use `max` instead of branching. You can add a code comment above the declaration of `salary` e.g. `// The number of credits paid daily (minimum 0).`